internal/trust/trust.go

```diff
@@ -0,0 +1,22 @@
+// package trust provides trusted certificate management.
+package trust
+
+// Certifier defines the contract to manage digital certificates in Kyma CLI.
+type Certifier interface {
+
+	// Certificate provides the decoded Kyma root certificate.
+	Certificate() ([]byte, error)
+
+	// StoreCertificate imports the given certificate file into the trusted root certificates manager of the OS.
+	StoreCertificate(file string, info Informer) error
+
+	// Instructions provides instructions on how to manually store a certificate.
+	// Use in case it can not be stored by calling StoreCertificate.
+	Instructions() string
+}
+
+// informer defines the war ceertification management informs about its progress.
```

Comment:
I know it's only a comment but something seems off - is this sentence correct?

Reply:
True, there are a couple of typos in the godoc of this interface, thanks for catching that one.

Reply:
Fixed thanks!